cicecore/cicedynB/general/ice_flux.F90

```diff
@@ -56,12 +56,14 @@ module ice_flux
 
        ! out to ocean          T-cell (kg/m s^2)
        ! Note, CICE_IN_NEMO uses strocnx and strocny for coupling
-         strocnxT, & ! ice-ocean stress, x-direction at T points, per ice fraction
-         strocnyT    ! ice-ocean stress, y-direction at T points, per ice fraction
+         strocnxT_sf, & ! ice-ocean stress, x-direction at T points, per ice fraction (scaled flux)
```

Comment:
The "scaled flux" terminology is confusing.  Scaled how?  "per ice fraction" means that the quantity is the mean (of whatever) over just the ice fraction of the grid cell.  That should be enough information, but if it's not clear, then say "averaged over the ice fraction".  This comment implies that the stress terms have been divided by the ice fractional area, so that they are not averaged over the full grid cell area.  Instead of _sf, maybe use _ice (although that's confusing too, with the history output standard of using _ai to mean the quantity includes a multiplicative factor of aice)